.buildkite/pipeline.yml

```diff
@@ -24,4 +24,6 @@ steps:
 
   - label: ':hammer: tests'
     commands:
+      - 'ln -s /var/lib/fc-ci/vmlinux.bin vmlinux'
```

Comment:
Since you support a `KERNELIMAGE` environment variable, you might as well skip this and just point that variable at `/var/lib/fc-ci/vmlinux.bin`